examples/manipulating_visualizing/plot_demo_glass_brain.py

```diff
@@ -20,10 +20,21 @@
 # demo glass brain plotting
 from nilearn import plotting
 
-plotting.plot_glass_brain(localizer_tmap_filename, threshold=3)
```

Comment:
This file is referenced from the front page. For this reason, we would like to keep this file as short as possible.

plot_demo_plotting.py or plot_demo_more_plotting.py would be better places to add this functionality. You could link to them in the docstring or plot_demo_glass_brain.py for users to more easily discover the cool options.